### ISSUE TITLE
Possible bug: nothing shown on Reddit tab, by default

### ISSUE THREAD

**Issue by AndroidDeveloperLB** [Alaory/WallMe-Wallpaper#33]
Is this intentional? v2.7

![image](https://user-images.githubusercontent.com/5357526/226914773-229e4b1c-e064-4b63-93ff-7a8f5bee03c0.png)

**Comment by Alaory** [comment]
yeah i made that the default. i thought it would be better to remove all default subreddits

**Comment by AndroidDeveloperLB** [comment]
OK I thought you should know.
Perhaps you should add a placeholder for empty state, with a button to reach the settings and add subreddits
